packages/devtools_app/lib/src/analytics/prompt.dart

```diff
@@ -4,44 +4,45 @@
 
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
 
 import '../common_widgets.dart';
 import '../config_specific/launch_url/launch_url.dart';
 import '../theme.dart';
 import 'provider.dart';
 
 /// Conditionally displays a prompt to request permission for collection of
-/// usage analytics.
+/// usage analytics and manages initializing GTags analytics.
```

Comment:
seems fragile to have the prompt responsible for initializing gtags. can we ignore that?

Reply:
moved the initialization code into the provider (now renamed to controller to be consistent with DevTools naming as a whole)